src/github/tests/test_utils.rs

```diff
@@ -267,23 +433,37 @@ impl GithubRead for GithubMock {
             .collect())
     }
 
-    fn repo(&self, _org: &str, _repo: &str) -> anyhow::Result<Option<Repo>> {
-        todo!()
+    fn repo(&self, org: &str, repo: &str) -> anyhow::Result<Option<Repo>> {
+        assert_eq!(org, DEFAULT_ORG);
+        Ok(self.repos.get(repo).cloned())
     }
 
-    fn repo_teams(&self, _org: &str, _repo: &str) -> anyhow::Result<Vec<RepoTeam>> {
-        todo!()
+    fn repo_teams(&self, org: &str, repo: &str) -> anyhow::Result<Vec<RepoTeam>> {
+        assert_eq!(org, DEFAULT_ORG);
+        Ok(self
+            .repo_members
+            .get(repo)
+            .cloned()
+            .map(|(teams, _)| teams)
+            .unwrap_or_default())
     }
 
-    fn repo_collaborators(&self, _org: &str, _repo: &str) -> anyhow::Result<Vec<RepoUser>> {
-        todo!()
+    fn repo_collaborators(&self, org: &str, repo: &str) -> anyhow::Result<Vec<RepoUser>> {
+        assert_eq!(org, DEFAULT_ORG);
+        Ok(self
+            .repo_members
+            .get(repo)
+            .cloned()
+            .map(|(_, members)| members)
+            .unwrap_or_default())
     }
 
     fn branch_protections(
         &self,
-        _org: &str,
+        org: &str,
         _repo: &str,
     ) -> anyhow::Result<HashMap<String, (String, BranchProtection)>> {
-        todo!()
+        assert_eq!(org, DEFAULT_ORG);
+        Ok(HashMap::default())
```

Comment:
are we returning this just momentarely? If yes, should we write a `// TODO` or better create a github issue?